ios/CodePush/CodePush.m

```diff
@@ -208,45 +208,64 @@ + (void)ensureBinaryBundleExists
 {
     if (![self binaryBundleURL]) {
         NSString *errorMessage;
-        
+
 #if TARGET_IPHONE_SIMULATOR
         errorMessage = @"React Native doesn't generate your app's JS bundle by default when deploying to the simulator. "
         "If you'd like to test CodePush using the simulator, you can do one of three things depending on your React "
         "Native version and/or preferred workflow:\n\n"
-        
+
         "1. Update your AppDelegate.m file to load the JS bundle from the packager instead of from CodePush. "
         "You can still test your CodePush update experience using this workflow (debug builds only).\n\n"
-        
+
         "2. Force the JS bundle to be generated in simulator builds by removing the if block that echoes "
         "\"Skipping bundling for Simulator platform\" in the \"node_modules/react-native/packager/react-native-xcode.sh\" file.\n\n"
-        
+
         "3. Deploy a release build to the simulator, which unlike debug builds, will generate the JS bundle (React Native >=0.22.0 only).";
 #else
         errorMessage = [NSString stringWithFormat:@"The specified JS bundle file wasn't found within the app's binary. Is \"%@\" the correct file name?", [bundleResourceName stringByAppendingPathExtension:bundleResourceExtension]];
 #endif
-        
+
         RCTFatal([CodePushErrorUtils errorWithMessage:errorMessage]);
     }
 }
 
 - (instancetype)init
 {
     self = [super init];
-    
+
     if (self) {
         [self initializeUpdateAfterRestart];
     }
-    
+
     return self;
 }
 
+- (void)clearDebugUpdates
+{
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
This same technique has been used in the loadBundle method in production with no issues. I am pretty sure it will be set because the bridge is initialized serially but i can double check